README.md

```diff
@@ -59,4 +59,10 @@ optional:
 
 -   **Warning**: If your project or this project is facing problems with dependencies, try 
     `npm audit fix` first please! The reason is because `npm audit` could have you running
-    code that changes your npm config instead of the local `package.json`.
\ No newline at end of file
+    code that changes your npm config instead of the local `package.json`.
+
+## Run on lambda
+-   To run this on lambda, you need to put the `app` folder, production `node_modules` 
```

Comment:
In my opinion we should assume any reader is computer-literate enough to know how to create a Zip archive with particular contents, so we can just say "[you need to] create a Zip archive containing the `app` folder, production `node_modules` folder, `.env`, `config.js`, `index.js`, `package.json`, and `package-lock.json`."